data/level_to_cpm.json

```diff
@@ -0,0 +1,81 @@
+{
+  "1":    0.094,
```

Comment:
Please don't use these CPM. Better use cp_multiplier and additional_cp_multiplier given from APIs instead.

TotalCPM = cp_multiplier + additional_cp_multiplier